actionview/lib/action_view/template.rb

```diff
@@ -9,8 +9,6 @@ module ActionView
   class Template
     extend ActiveSupport::Autoload
 
-    mattr_accessor :finalize_compiled_template_methods, default: true
-
```

Comment:
Public API. Deprecation message?

Reply:
Deprecated in 88b3c454f753c0ce85e8df01d5aacf70fcf29b91